internal/compiler/widgets/common/spinbox-base.slint

```diff
@@ -87,6 +87,11 @@ export component SpinBoxBase {
             return reject;
         }
 
+        double-clicked => {
```

Comment:
This fixes that particular issue, but it makes me wonder: There are more click behaviours on the underlying `TextInput` that should be exposed, but aren't - such as triple click. And context menu in the future.

I think that perhaps this `TouchArea` should be placed underneath the `TextInput` and only deal with the scroll event. The existing `clicked` and the new `double-clicked` should already be handled by `TextInput` natively, no?